Intersect.Server.Core/Entities/Player.cs

```diff
@@ -5438,36 +5437,16 @@ public void UseSpell(int spellSlot, Entity target)
                 // retargeting for auto self-cast on friendly when targeting hostile
                 CastTarget = target;
 
-                //Check if the caster has the right ammunition if a projectile
-                if (spell.SpellType == SpellType.CombatSpell &&
-                    spell.Combat.TargetType == SpellTargetType.Projectile &&
-                    spell.Combat.ProjectileId != Guid.Empty)
-                {
-                    var projectileBase = spell.Combat.Projectile;
-                    if (projectileBase != null && projectileBase.AmmoItemId != Guid.Empty)
-                    {
-                        TryTakeItem(projectileBase.AmmoItemId, projectileBase.AmmoRequired);
-                    }
-                }
-
                 if (spell.CastAnimationId != Guid.Empty)
                 {
                     PacketSender.SendAnimationToProximity(
                         spell.CastAnimationId, 1, base.Id, MapId, 0, 0, Dir, MapInstanceId
                     ); //Target Type 1 will be global entity
                 }
 
-                // Check if the player isn't casting a spell already.
-                if (!IsCasting)
+                //Tell the client we are channeling the spell
+                if (IsCasting)
                 {
-                    // Player is not casting a spell, cast now!
-                    CastTime = 0;
-                    CastSpell(Spells[SpellCastSlot].SpellId, SpellCastSlot);
-                    CastTarget = null;
-                }
-                else
-                {
-                    //Tell the client we are channeling the spell
```

Comment:
Explanation for removal: duplicated logic, we were already doing this here:

https://github.com/AscensionGameDev/Intersect-Engine/blob/116dd44e9260807fba89ac928a9f240849b6e57e/Intersect.Server.Core/Entities/Entity.cs#L314-L320